setup.py

```diff
@@ -78,6 +78,12 @@
 package_info['package_data'].setdefault(PACKAGENAME, [])
 package_info['package_data'][PACKAGENAME].append('data/*')
 
+# Define entry points for command-line scripts
+entry_points = {}
+entry_points['console_scripts'] = [
+    'astropy-package-template-example = packagename.example_mod:main',
+]
```

Comment:
Not really ... If you find it simpler when combined into one statement I'll make the change.